platform/http_request.cpp

```diff
@@ -386,6 +387,12 @@ HttpRequest::~HttpRequest()
 {
 }
 
+HttpRequest::ProgressT const & HttpRequest::Progress() const
+{
+  ASSERT(m_progress.first <= m_progress.second, ());
```

Comment:
Как ты предлагаешь это сделать для файлов с неизвестным размером, не ломая апишку?

> On Apr 6, 2016, at 14:12, Илья Гречухин <notifications@github.com> wrote:
> 
> In platform/http_request.cpp <https://github.com/mapsme/omim/pull/2756#discussion_r58687769>:
> 
> > @@ -386,6 +387,12 @@ HttpRequest::~HttpRequest()
> >  {
> >  }
> >  
> > +HttpRequest::ProgressT const & HttpRequest::Progress() const
> > +{
> > +  ASSERT(m_progress.first <= m_progress.second, ());
> Очевидно, сделать m_progress.second >= m_progress.first.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2756/files/193bf11a2e6f8a3ead0d909e7f91c3f351ef2e0a#r58687769>